### PR TITLE
bump github.com/aws/aws-sdk-go from 1.44.83 to 1.44.317

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

**Review by richardcase — APPROVED** [review]

**Review by furkatgofurov7 — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,20 +1,63 @@
 module github.com/rancher/eks-operator
 
-go 1.16
+go 1.19
```
Comment (furkatgofurov7) [inline go.mod:3]: go 1.19 EOL with 1.21 being out, let's bump it to 1.20 if possible

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (mjura) [reply go.mod]: done, bumped to 1.20

**Comment by furkatgofurov7** [comment]
CI is failing due to:

```
go: github.com/rancher/lasso@v0.0.0-20221227210133-6ea88ca2fbcc requires
	github.com/matttproud/golang_protobuf_extensions@v1.0.2-0.20181231171920-c182affec369: missing go.sum entry; to add it:
	go mod download github.com/matttproud/golang_protobuf_extensions
time="2023-08-15T06:07:16Z" level=fatal msg="exit status 1"
```

**Comment by mjura** [comment]
There was several backports needed from branch `release-v2.8`

**Comment by furkatgofurov7** [comment]
> There was several backports needed from branch `release-v2.8`

@mjura this is targeting operator branch release-v2.7, should not it be release-v2.8?

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Pull request merged**